pkgs/tools/misc/ufetch-nixos/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "ufetch-nixos";
+  src = ./ufetch-nixos;
+
+  phases = "installPhase";
```

Comment:
This needs to be a list but it is not recommended to remove phases this way because it removes more than necessary. In this case you are missing `fixupPhase` so `patchShebangsAuto` is not run. See https://github.com/NixOS/nixpkgs/issues/28910.